Makefile

```diff
@@ -1,12 +1,15 @@
 # $FreeBSD$
 
+.PATH: ${.CURDIR}/lib
+
 MAN=
-CFLAGS=	-Wall -lmd -lz -lfetch
+CFLAGS+= -Wall
+LDFLAGS= -lmd -lz -lfetch
 
 PROG=	ogit
 
-SRCS=	ogit.c lib/ini.c lib/index.c lib/common.c lib/pack.c remote.c init.c \
-	lib/zlib-handler.c lib/buffering.c lib/loose.c \
+SRCS=	ogit.c ini.c index.c common.c pack.c remote.c init.c \
```

Comment:
Any objection to splitting the lib/ dependencies out onto a separate SRCS+= line while you're at it? I'd like to see lib/ get turned into an actual lib at some point in the near future.

Reply:
Done.